.github/workflows/doc.yml

```diff
@@ -0,0 +1,24 @@
+name: Sync OAS to ReadMe
+on:
+  push:
+    branches:
+      - main
+      - readme-sync
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: readmeio/github-readme-sync@v2
+        with:
+          readme-oas-key: ${{ secrets.README_TOKEN }}
```

Comment:
@atymchuk could you please check if this token is correct.

Reply:
Not really - the tokens are only known at the time they are posted. If @BupycHuk grabbed it from the readme.io's site it should be OK. This run indirectly confirms it's OK, so should be good to go.